meshnets/tune.py

```diff
@@ -72,10 +74,23 @@ def main(_):
     if random_seed is not None:
         torch.manual_seed(random_seed)
 
+    dataset_name = os.path.basename(FLAGS.data_dir)
     dataset = FromDiskGeometricDataset(FLAGS.data_dir)
     train_dataset, validation_dataset = random_split(
         dataset, [FLAGS.train_split, FLAGS.validation_split])
 
+    val_datasets_names = [dataset_name]
+    val_datasets = [validation_dataset]
+
+    for val_data_dir in FLAGS.val_data_dirs:
+        dataset_name = os.path.basename(val_data_dir)
+        dataset = FromDiskGeometricDataset(val_data_dir)
+        _, validation_dataset = random_split(
+            dataset, [FLAGS.train_split, FLAGS.validation_split])
```

Comment:
Why do we split the data. I was already expecting that the directories in `val_data_dirs` to contain exactly the data that we wanted to use for validation.

Reply:
`val_data_dir` gives the path to the datasets we want to use for validation, we still split them to avoid evaluating on the whole dataset